src/core_plugins/console/public/src/input.js

```diff
@@ -13,232 +13,238 @@ let es = require('./es');
 
-input.commands.addCommand({
-  name: 'auto indent request',
-  bindKey: {win: 'Ctrl-I', mac: 'Command-I'},
-  exec: function () {
-    input.autoIndent();
-  }
-});
-input.commands.addCommand({
-  name: 'move to previous request start or end',
-  bindKey: {win: 'Ctrl-Up', mac: 'Command-Up'},
-  exec: function () {
-    input.moveToPreviousRequestEdge()
-  }
-});
-input.commands.addCommand({
-  name: 'move to next request start or end',
-  bindKey: {win: 'Ctrl-Down', mac: 'Command-Down'},
-  exec: function () {
-    input.moveToNextRequestEdge()
-  }
-});
-
-
-/**
- * COPY AS CURL
- *
- * Since the copy functionality is powered by a flash movie (via ZeroClipboard)
- * the only way to trigger the copy is with a litteral mouseclick from the user.
- *
- * The original shortcut will now just open the menu and highlight the
- *
- */
-var $copyAsCURL = $('#copy_as_curl');
-var zc = (function setupZeroClipboard() {
-  var zc = new ZeroClipboard($copyAsCURL); // the ZeroClipboard instance
-
-  zc.on('wrongflash noflash', function () {
-    if (!storage.get('flash_warning_shown')) {
-      alert('Console needs flash version 10.0 or greater in order to provide "Copy as cURL" functionality');
-      storage.set('flash_warning_shown', 'true');
+  input.autocomplete = new Autocomplete(input);
+
+  input.$actions = $actionsEl; // $("#editor_actions");
+
+  input.commands.addCommand({
+    name: 'auto indent request',
+    bindKey: {win: 'Ctrl-I', mac: 'Command-I'},
+    exec: function () {
+      input.autoIndent();
     }
-    $copyAsCURL.hide();
   });
-
-  zc.on('ready', function () {
-    function setupCopyButton(cb) {
-      cb = typeof cb === 'function' ? cb : $.noop;
-      $copyAsCURL.css('visibility', 'hidden');
-      input.getRequestsAsCURL(function (curl) {
-        $copyAsCURL.attr('data-clipboard-text', curl);
-        $copyAsCURL.css('visibility', 'visible');
-        cb();
-      });
+  input.commands.addCommand({
+    name: 'move to previous request start or end',
+    bindKey: {win: 'Ctrl-Up', mac: 'Command-Up'},
+    exec: function () {
+      input.moveToPreviousRequestEdge()
     }
-
-    input.$actions.on('mouseenter', function () {
-      if (!$(this).hasClass('open')) {
-        setupCopyButton();
-      }
-    });
   });
-
-  zc.on('complete', function () {
-    $copyAsCURL.click();
-    input.focus();
+  input.commands.addCommand({
+    name: 'move to next request start or end',
+    bindKey: {win: 'Ctrl-Down', mac: 'Command-Down'},
+    exec: function () {
+      input.moveToNextRequestEdge()
+    }
   });
 
-  return zc;
-}());
 
-/**
- * Setup the "send" shortcut
- */
+  /**
+   * COPY AS CURL
+   *
+   * Since the copy functionality is powered by a flash movie (via ZeroClipboard)
+   * the only way to trigger the copy is with a litteral mouseclick from the user.
+   *
+   * The original shortcut will now just open the menu and highlight the
+   *
+   */
+  var $copyAsCURL = $copyAsCurlEl; // $('#copy_as_curl');
```

Comment:
Actually, is this variable needed? Can we just refer to `$copyAsCurlEl` wherever we refer to `$copyAsCurl`?